libs/wire-subsystems/src/Wire/UserStore.hs

```diff
@@ -54,6 +54,7 @@ data UserStore m a where
   GetIndexUsersPaginated :: Int32 -> Maybe PagingState -> UserStore m (PageWithState IndexUser)
   GetUsers :: [UserId] -> UserStore m [StoredUser]
   UpdateUser :: UserId -> StoredUserUpdate -> UserStore m ()
+  UpdateEmailUnvalidated :: UserId -> EmailAddress -> UserStore m ()
```

Comment:
do we need a DeleteEmailUnvalidated?  or does this happen via TTL?  or implicitly when something else happens?

Reply:
ah, see `updateEmailAndDeleteEmailUnvalidate` and other places.  we do delete unvalidated emails, just not in the subsystems yet.